static/src/javascripts/projects/commercial/modules/dfp/define-slot.js

```diff
@@ -97,53 +97,62 @@ const defineSlot = (adSlotNode: Element, sizes: Object): Object => {
     }
 
     /*
-        The iasOptimisation implementation below is still a WiP, hence the
-        console.log statements and the dev-only execution.
+        For each ad slot defined, we request information from IAS, based
+        on slot name, ad unit and sizes. We then add this targeting to the
+        slot prior to requesting it from DFP.
+
+        We race the request to IAS with a Timeout of 2 seconds. If the
+        timeout resolves before the request to IAS then the slot is defined
+        without the additional IAS data.
      */
 
-    if (config.switches.iasOptimisation && config.get('page.isDev', false)) {
-        console.log(`Defining ${id}`);
-        /* eslint-disable no-underscore-dangle, no-multi-assign */
-        const iasPET = (window.__iasPET = window.__iasPET || {});
-        /* eslint-enable no-underscore-dangle, no-multi-assign */
+    if (config.switches.iasOptimisation) {
```

Comment:
Better written as `config.get('switches.iasOptimisation')`, because it's typesafe.